CoinbasePro/WebSocket/WebSocket.cs

```diff
@@ -226,7 +261,7 @@ public void WebSocket_Closed(object sender, EventArgs e)
         private List<Channel> GetChannels()
         {
             var channels = new List<Channel>();
-            if (channelTypes == null || channelTypes.Any())
+            if (channelTypes == null || channelTypes.Count == 0)
```

Comment:
lets keep this consistent with above. `!channelTypes.Any()`

Reply:
I have implemented the changes.  Git now has conflict after I changed `if (channelTypes == null || channelTypes.Count == 0)` to `(channelTypes == null || !channelTypes.Any())`